data/iidx.tsv

```diff
@@ -26,3 +26,10 @@
 26037		CANVAS feat. Quimär	
 26070	Xlø		
 26101	Geirskögul		
+27083		DÉ DÉ MOUSE	
+27004	Dans la nuit de l'eternite		
+27035		CANVAS feat. Quimär	
+28007		CANVAS feat. Quimär & BEMANI Sound Team "L.E.D.-G"	
+28017	Pārvatī		
+28050	POLꞰAMAИIA		
+28084	Ignis†Iræ	
```

Comment:
Another tab needs to be inserted at the end of the line.

Reply:
Gonna be honest, I underestimated just how much my ability to work on this would drop once I started working a "real" job. Thanks for the notes though. If you wanted to take over the pull request you would have my thanks tbh.